examples/complete-guide.ipynb

```diff
@@ -686,37 +686,29 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 25,
-   "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "[(0, 1), (0, 2), (0, 3), (0, 4), (1, 0), (1, 2), (1, 3), (1, 4), (2, 0), (2, 1), (2, 3), (2, 4), (3, 0), (3, 1), (3, 2), (3, 4), (4, 0), (4, 1), (4, 2), (4, 3)]\n"
-     ]
-    }
-   ],
-   "source": [
-    "#num_users = 5\n"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 33,
+   "execution_count": 17,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
       "These are the current metrics:\n",
-      "[<trecs.metrics.measurement.MSEMeasurement object at 0x7ff52f384950>, <trecs.metrics.measurement.HomogeneityMeasurement object at 0x7ff52fcaaf50>]\n"
+      "[<trecs.metrics.measurement.MSEMeasurement object at 0x7fc699c71290>, <trecs.metrics.measurement.HomogeneityMeasurement object at 0x7fc6b149f6d0>]\n"
      ]
     }
    ],
    "source": [
-    "filtering = ContentFiltering(num_users=50, num_items=100, num_attributes=10)\n",
+    "#change the number of items and users to make metric values more reasonable and dimensions distinguishable\n",
+    "number_of_items=100\n",
+    "number_of_users=50\n",
+    "number_of_attributes=20\n",
+    "\n",
+    "#change the distribution of item attributes so average feature score range can be added later\n",
```

Comment:
Yeah, the non-binary features thing does break the Chaney assumptions, but luckily our implementation works when there aren't binary item attributes. Since it's perfectly "allowed" in t-recs to run the simulation with real-valued item attributes, this is fine!